lib/l10n/l10n.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+//localisation class used for language change
+
+class L10n{
```

Comment:
Wäre cool, wenn der Klassenname auch mit dem Dateinamen übereinstimmt.
Könntest du da quasi die Datei noch in _L10n.dart_ umbenennen?